### PR TITLE
5xx error link syntax fix

### PR DESCRIPTION
**Pull request by Smullz622**

### CODE REVIEW

**Review by whereismyjetpack — DISMISSED** [review]

**Comment by banukutlu** [comment]
CC shows a big coverage change 👀

File: config/locales/blacklight.en.yml [hunk 1]
```diff
@@ -15,7 +15,7 @@ en:
       sms: 'SMS'
     search:
       errors:
-        request_error: "Sorry, our search index experienced a problem. Please try again in a moment. If this error persists, please [report the issue](https://libraries.psu.edu/website-feedback) to Libraries Strategic Technology."
+        request_error: "Sorry, our search index experienced a problem. Please try again in a moment. If this error persists, please <a href="https://libraries.psu.edu/website-feedback">report the issue</a> to Libraries Strategic Technology."
```
Comment (banukutlu) [inline config/locales/blacklight.en.yml:18]: There are double quotes inside double quotes here, test is failing with syntax error. Maybe sth like below? Of course needs to update references.

```suggestion
        request_error: 
          html: 'Sorry, our search index experienced a problem. Please try again in a moment. If this error persists, please <a href="https://libraries.psu.edu/website-feedback">report the issue</a> to Libraries Strategic Technology.'
```

**Review by banukutlu — CHANGES_REQUESTED** [review]

**Review by ajkiessl — APPROVED** [review]
Not sure how to test this, but the syntax looks good to me.  I think the translation field either needs to be called "html" or end with "_html" for the html to render properly, so I think this should be good.

**Review by banukutlu — APPROVED** [review]
thanks @Smullz622

**Pull request merged**